pkg/dbutils/dbutils.go

```diff
@@ -14,6 +14,26 @@ const (
 	lockTimeoutCode = "55P03"
 )
 
+type Lsn uint64
+
+func (l Lsn) String() string {
+	return pgx.FormatLSN(uint64(l))
+}
+
+func (l *Lsn) Parse(lsn string) error {
+	tmp, err := pgx.ParseLSN(lsn)
+	if err != nil {
+		return err
+
+	}
+	*l = (Lsn)(tmp)
+	return nil
+}
+
+const InvalidLSN Lsn = 0
```

Comment:
if we use `Lsn` in that writing, then let's use it like here as well -> `InvalidLsn`